File: back-end/ReportingTasksPhp/Models/WorkerToProject.php
<?php


class WorkerToProject implements JsonSerializable{
  public $WorkerToProjectId;
    public $UserId;
    public $ProjectId;
    public $Hours;
     public function __construct($sqlRaw_) {    
         $this->WorkerToProjectId = $sqlRaw_['worker_to_project_id'];
         $this->UserId = $sqlRaw_['user_id']; 
         $this->ProjectId = $sqlRaw_['project_id']; 
         $this->Hours = $sqlRaw_['hours'];   
    }
    public function jsonSerialize() {
        return get_object_vars($this);
    }   
}

File: back-end/ReportingTasksProject/01_BOL/WorkerToProject.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BOL
{
   public class WorkerToProject
    {
       
        public int WorkerToProjectId { get; set; }
        [Required]
        public int UserId { get; set; }
        [Required]
        public int ProjectId { get; set; }
        [Range(0, int.MaxValue, ErrorMessage = "Please enter valid integer Number")]
        public int Hours { get; set; }
    }
}
